bin/build-harmony

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# build the cmr granule locator task image
+# cd tasks/query-cmr
+# docker build --tag query-cmr:latest .
+# cd ../..
+
+# build the harmony-local docker image
+docker build --tag harmony/harmony-local:latest .
```

Comment:
Wonder if we should just publish all this to DockerHub

Reply:
That would eliminate the build step for people just wanting to run Harmony.